controllers/admin/admin_media_controller.py

```diff
@@ -26,6 +26,21 @@ def get(self):
         self.response.out.write(template.render(path, self.template_values))
 
 
+class AdminMediaDeleteReference(LoggedInHandler):
+    def post(self, media_key_name):
+        self._require_admin()
+
+        media = Media.get_by_id(media_key_name)
+
+        media.references.remove(media.create_reference(
+            self.request.get("reference_type"),
+            self.request.get("reference_key_name")))
+
+        media.put() ## Todo, update manipulators to support removing an item that ends a list
```

Comment:
"When I delete the reference here, then go to do a MediaManipulator.createOrUpdate(), the "old model" the MediaManipulator is getting back from its internal ndb call already has references = [], even though the only place that mutation has happened is against the media object I am attempting to write back to the database."

I believe this is the case due to instance caching. It's the same reason we have `mode.dirty = True  # this is hacky -greg` in some places.